core/deployment/src/main/java/io/quarkus/deployment/steps/ConfigGenerationBuildStep.java

```diff
@@ -759,6 +759,12 @@ private static Set<String> staticSafeServices(Set<String> services) {
         ClassLoader classloader = Thread.currentThread().getContextClassLoader();
         Set<String> staticSafe = new HashSet<>();
         for (String service : services) {
+            // SmallRye Config services are always safe, but they cannot be annotated with @StaticInitSafe
+            if (service.startsWith("io.smallrye.config")) {
```

Comment:
Sorry for the nitpicking but please add a `.` at the end.

You never know what else will end up in SmallRye and I would prefer avoiding bugs because we end up having services with `io.smallrye.configurator` or whatever.

Reply:
Sure.